src/libraries/System.Private.CoreLib/src/System/Runtime/Serialization/SerializationInfo.SerializationGuard.cs

```diff
@@ -47,7 +48,7 @@ public static bool DeserializationInProgress
 
         // Throws a SerializationException if dangerous deserialization is currently
         // in progress
-        public static void ThrowIfDeserializationInProgress()
+        internal static void ThrowIfDeserializationInProgress()
```

Comment:
Right now, all methods on this type accessed from elsewhere are public. Is there a value in making a few of them internal? It seems less consistent.

Reply:
Not everything is public there but I removed the change